osquery/events/windows/windowseventlogparser.cpp

```diff
@@ -18,7 +18,61 @@
 #include <osquery/logger/logger.h>
 #include <osquery/utils/conversions/windows/strings.h>
 
+namespace pt = boost::property_tree;
```

Comment:
If we are adding `pt` as an alias, can we update all the existing usage of `boost::property_tree` in this file to use `pt`? I'd rather use one or the other, but not mix both.